granulate_utils/linux/cgroups/cgroup.py

```diff
@@ -98,3 +99,67 @@ def get_cgroup_mount(controller: str, resolve_host_root_links: bool = True) -> O
 
 def is_known_controller(controller: str) -> bool:
     return controller in SUBSYSTEMS
+
+
+def get_cgroup_relative_path(subsystem: str) -> str:
+    """
+    Get the subsystem cgroup path relative to the cgroup sysfs path
+    :param subsystem: the subsystem
+    :return: the relative path, with a '/' prefix
+    """
+    hierarchy_details = get_cgroups()
+    for line in hierarchy_details:
+        if subsystem in line[1]:
+            return line[2]
+    raise Exception(f"{subsystem!r} not found")
+
+
+class CgroupUtils:
```

Comment:
Why not define all the methods top-level?

Reply:
:+1: , no value for classes if it's functional after all.